src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -31,13 +34,48 @@ const ConnectedFilterList = connect(
   })
 )(FiltersList);
 
-export default () =>
-  <Filters>
+// tslint:disable-next-line:variable-name
+const ConnectedPrintButton = connect(
+  (state: AppState) => ({
+    disabled: selectors.summaryIsLoading(state),
+    loading: selectors.summaryIsLoading(state),
+    shouldFetchMore: selectors.hasMoreResults(state),
+  }),
+  (dispatch: Dispatch) => ({
+    loadHighlightsAndPrint: flow(printStudyGuides, dispatch),
+  }),
```

Comment:
It follows the same schema as `./highlights/.../Filters`

Reply:
we're following a container pattern, it could be implemented as 

```
import ReactComponent from 'place';

export ContainerComponent = () => <ReactComponent
  disabled={useSelector(selectors.summaryIsLoading)}
/>;

// OR

export ContainerComponent = connect(
  (state) => ({
    disabled: selectors.summaryIsLoading(state)
  })
)(ReactComponent)
```

they're basically the same, connect has some optimizations and is the way that i've done this in the past, but i guess it probably doesn't really matter either way.